chameleonultragui/lib/main.dart

```diff
@@ -260,13 +260,13 @@ class _MainPageState extends State<MainPage> {
                           destinations: [
                             // Sidebar Items
                             NavigationRailDestination(
-                              icon: const Icon(Icons.home),
+                              icon: const Icon(Icons.home_outlined),
```

Comment:
id stick to the non outlined ones on the sidebar, but other than that, sure, lgtm